var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -90,7 +91,7 @@ def libs(self):
 
             return find_libraries(
                 libraries, root=self.prefix,
-                shared=False, recursive=True
+                shared=True, recursive=True
```

Comment:
This should depend on the value of `'+shared' in self.spec`, the package is `~shared` by default and it looks like it can't even build shared in recent versions.